lib/TreeSearch.js

```diff
@@ -21,14 +21,12 @@ TreeSearch.postOrder = function(simplificationFunction) {
 // A helper function for performing a tree search with a function
 function search(simplificationFunction, node, preOrder) {
   let status;
-
```

Comment:
can you add these spaces back in? I like them for code readability, and they're also changing files you didn't change otherwise which is confusing to include in this PR for future investigators :)

(same for the other deleted and added lines - you can see where they are below in the diff)